public/js/controllers/charts_controller.js

```diff
@@ -29,6 +30,22 @@ function intComma (amount) {
   return amount.toLocaleString(undefined, { maximumFractionDigits: 0 })
 }
 
+function formatHashRate (value, displayType) {
+  value = parseInt(value)
+  if (value <= 0) return value
+  var shortUnits = ['Th', 'Ph', 'Eh']
+  var labelUnits = ['TeraHash/s', 'PentaHash/s', 'ExaHash/s']
```

Comment:
penta -> peta

Reply:
The Hash part does not need caps.  Actually, I'm not sure should be any caps (terahash/s)